chainer/training/extensions/computational_graph.py

```diff
@@ -16,7 +17,32 @@ def dump_graph(root_name, out_name='cg.dot',
     This extension dumps a computational graph. The graph is output in DOT
     language.
 
-    It only dumps a graph at the first iteration by default.
+    It only dumps a graph at the first invocation.
+
+    .. note::
+       As of v2.0.0, the computational graph is not kept by default. This
+       extension changes this behavior until the first invocation. **It is
+       strongly recommended to use it with the default trigger setting.**
+
+       The detailed behavior of this extension since v2.0.0 is as follows.
+
+       1. In the initializer, it turns on the
```

Comment:
I feel "initialization" is a little bit ambiguous because it sounds like `__init__`. Maybe the naming of the API is already ambiguous a bit, but I think "initializer" refers to the `initializer` option of `make_extension` and this usage of the term is consistent.